zebra-chain/src/transaction/sighash.rs

```diff
@@ -48,6 +52,23 @@ impl HashType {
     }
 }
 
+/// A Signature Hash (or SIGHASH) as specified in
+/// https://zips.z.cash/protocol/protocol.pdf#sighash
+#[derive(Copy, Clone, Eq, PartialEq, Hash, Serialize, Deserialize, Debug)]
+pub struct Hash(pub [u8; 32]);
+
+impl AsRef<[u8; 32]> for Hash {
+    fn as_ref(&self) -> &[u8; 32] {
+        &self.0
+    }
+}
+
+impl AsRef<[u8]> for Hash {
+    fn as_ref(&self) -> &[u8] {
+        &self.0
+    }
+}
+
```

Comment:
Is this not the same as the `blake2b_simd::Hash` being returned from https://doc-internal.zebra.zfnd.org/zebra_chain/transaction/enum.Transaction.html#method.sighash ?